### ISSUE TITLE
\begin{verbatim} generates deprecated \tt

### ISSUE THREAD

**Issue by brunom** [kosmikus/lhs2tex#44]
The following .lhs

```
\documentclass{scrbook}
%include lhs2TeX.fmt
\begin{document}
\begin{verbatim}
foo
\end{verbatim}
\end{document}
```

when processed by

`lhs2TeX --poly scratch.lhs -o scratch.tex && pdflatex scratch.tex`

generates the warning

```
Class scrbook Warning: Usage of deprecated font command `\tt'!
(scrbook)              You should note, that in 1994 font command `\tt' has
(scrbook)              been defined for compatiblitiy to Script 2.0 only.
(scrbook)              Now, after two decades of LaTeX2e and NFSS2, you
(scrbook)              shouldn't use such commands any longer and within
(scrbook)              KOMA-Script usage of `\tt' is definitely deprecated.
(scrbook)              See `fntguide.pdf' for more information about
(scrbook)              recommended font commands.
(scrbook)              Note also, that KOMA-Script will remove the definition
(scrbook)              of `\tt' anytime until release of about version 3.20.
(scrbook)              But for now, KOMA-Script will replace deprecated `\tt'
(scrbook)              by `\normalfont \ttfamily ' on input line 97.
```

**Comment by kosmikus** [comment]
True. These deprecation warnings exist for a long time, and in principle, I agree. I've just been admittedly too lazy to sort it out myself and make sure everything keeps working, because tt mode is not a mode of lhs2TeX that I use very often.

I'd be happy to accept a patch for this.

**Comment by brunom** [comment]
After upgrading from texlive2015 to texlive2016, the warning became an error.

**Comment by franklinyu** [comment]
Probably fixed by #45?

**Comment by kosmikus** [comment]
Should be fixed in master and 1.20.
